src/azure-cli/azure/cli/command_modules/acr/_utils.py

```diff
@@ -285,10 +286,14 @@ def get_custom_registry_credentials(cmd,
     :param str password: The password for custom registry (plain text or a key vault secret URI)
     :param str identity: The task managed identity used for the credential
     """
+    Credentials, SourceRegistryCredentials, CustomRegistryCredentials, SecretObject, \
+        SecretObjectType = cmd.get_models(
+            'Credentials', 'CustomRegistryCredentials', 'SourceRegistryCredentials', 'SecretObject',
+            'SecretObjectType',
+            operation_group='tasks')
```

Comment:
What does this change do?

Reply:
I think they are moving model imports to top of the function instead of them being in-lined in different parts of the function.
This is likely to make the code more readable / clear. @00Kai0 can clarify

Reply:
Like @adewaleo said, we just want to make the code more readable.

Reply:
Use `operation_group` defined in _shared.py to import models from different API versions.